src/modules/access-wallet/software/handlers/helpers.js

```diff
@@ -0,0 +1,15 @@
+import { isHexString } from 'ethereumjs-util';
+
+const SOFTWARE_WALLET_TYPES = [
+  'keystore',
+  'mnemonic',
+  'private-key',
+  'overview'
+];
```

Comment:
```
SOFTWARE_WALLET_TYPES = {
KEYSTORE:'keystore'....
}
```

Reply:
fixed in : https://github.com/MyEtherWallet/MyEtherWallet/pull/2846/commits/2f678616ff35a0e9dccfbdce859132cf5e069d58